tools/build_docker.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Seems like this file is not needed, its the same as for linux wo docker

Reply:
Actually it's not, it enables the `-DUSE_OPENMP=ON` flag in cmake

Reply:
Now there are bash files with openmp https://github.com/brainflow-dev/brainflow/blob/master/tools/build_linux_omp.sh and no need to rebuild this code in docker since it should be precompiled there